jib-plugins-extension-common/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'jib-plugins-extension-common'
```

Comment:
I feel like this file shouldn't be here.

Reply:
I still don't know what this file is for. Is this required for releasing? We need to publish this sub-project.